index.html

```diff
@@ -1 +1,42 @@
-My Page
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Our app has 2 views. So remove the status view from index and add it to a partial. and then load the partial using routesindex